code/modules/events/dust.dm

```diff
@@ -1,10 +1,10 @@
 /datum/round_event_control/meteor_wave/dust
 	name = "Minor Space Dust"
 	typepath = /datum/round_event/meteor_wave/dust
-	weight = 200
+	weight = 50		// A lot, but not so much that it’s a constant hazard.

```

Comment:
I have a feeling that if you lower the weight of space dust, that will, by proxy, increase the chances of everything else.